.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  # Using this mirror lets us use mypyc-compiled black, which is about 2x faster
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 23.9.1
+    hooks:
+      - id: black
+        args: [--check, --diff]
+        # It is recommended to specify the latest version of Python
+        # supported by your project here, or alternatively use
+        # pre-commit's default_language_version, see
+        # https://pre-commit.com/#top_level-default_language_version
+        # language_version: python3.11
```

Comment:
Why is this commented out?

Reply:
We don't have the max Python version pinned in setup.py, so why pin it here? Just another thing to have to keep updated. I expect when we get to Pymoca v1.0, we will want to pin a lot more things, including max Python version.

Reply:
Well, then the question is, why are you leaving the comment above the language-specification line

Reply:
> Well, then the question is, why are you leaving the comment above the language-specification line

Because I didn't want to remove the other comments that pre-commit generated and for future reference when we do pin versions. You make a good point, I should put in a comment to explain this.

Reply:
Makes sense, I thought it was copy-paste from some example, but if it's generated indeed just keep it